README.md

```diff
@@ -2,15 +2,32 @@
 
 Scalable Object Ledger and CDN.
 
-## Quick Start
+## Introducing Warehouse 7
+
+Warehouse 7, also known as Warehouse Next, is a scalabe object ledger and cdn system for powering dynamic web applications. The new system has been entirly redisgned using a lightweight, portable and flexible architecture.
+
+While designing the new system, it made sense to add a higher level of abstraction.
+The idea is expose a generic Object API and a CDN service. The user can use the CDN service to upload the desired assets while using the Object API to store the metadata of the assets (e.g., file urls, etc.)
+
+The Object API acts as generic ledger and it can be used by a third-party system to store any type of data.
```

Comment:
```suggestion
The Object API acts as generic ledger and it can be used to store any type of data. In particular, the Object API is good at storing versioned JSON data that may have one or more "head" version(s) for one or more "environment(s)"; for example, "production" and "staging".
```